pyproject.toml

```diff
@@ -33,7 +33,7 @@ docstring-convention = "google"
 
 [tool.pyright]
 include = ["ops/jujuversion.py", "ops/log.py", "ops/model.py", "ops/version.py",
-    "ops/__init__.py", "ops/framework.py", "opps/pebble.py"]
+    "ops/__init__.py", "ops/framework.py", "ops/pebble.py", "ops/charm.py"]
```

Comment:
oops.  Or should I say "opps".  How did we not have any pebble typing errors get introduced while this was like this?

Reply:
no idea :D I think pyright silently succeeds if a pattern doesn't match
and probably ops/pebble.py became opps when I switched the pyright from blacklist to whitelist, so the tests were green and nice because the tests did run until the very last moment